### ISSUE TITLE
Prevents kanban plugin from setting per-board lane widths

### ISSUE THREAD

**Issue by lachholden** [damiankorcz/Prism-Theme#138]
Per title. This seems to be the offending line:

https://github.com/damiankorcz/Prism-Theme/blob/d48c21a7fc444b1705ede0d9849589c35ec9d228/src/scss/Plugins/Community/kanban.scss#L12C3-L12C3

I have some boards setup where I need really wide widths for the cards to be useful. For now I'm overriding it in my custom CSS as follows, but it would be great if the Kanban plugin/board settings just continued to work.

```css
.kanban-plugin__lane-wrapper[data-hitboxid*="board name"] {
  width: 400px !important;
}
```

Cheers!

**Comment by damiankorcz** [comment]
Hi @lachholden,
I'll remove that line in the next update so you will be able to change it in Kanban settings. Pretty sure I've put it in because the width was something else and I wanted it to align nicely with the custom backgrounds I add in Prism. Seems that the value I use is the default in Kanban now so it all works out 😅

**Comment by damiankorcz** [comment]
Hi @lachholden,
This should now be fixed in Prism 3.4.0 (Currently available for the Insider release only). Changing the width in Kanban Settings page should now apply correctly.
